mne/io/artemis123/utils.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import os.path as op
+from ...utils import logger
+from ...transforms import rotation3d_align_z_axis
+
+
+def _load_mne_locs(fname=None):
+    """Load MNE locs structure from file (if exists) or recreate it."""
+    if (not fname):
```

Comment:
This is a test of whether `fname == None or fname == ''`
basically its a multiline default argument.
I can change it how ever you like
do you prefer
`if not fname:`
or
`if fname is None`